src/sql/workbench/contrib/query/browser/queryResultsEditor.ts

```diff
@@ -122,35 +127,55 @@ export class QueryResultsEditor extends EditorPane {
 	}
 
 	dispose() {
+		if (!this.styleSheet) {
+			return;
+		}
 		this.styleSheet.remove();
 		this.styleSheet = undefined;
 		super.dispose();
 	}
 
 	layout(dimension: DOM.Dimension): void {
+		if (!this.resultsView) {
```

Comment:
All of these calls can be simplified to be like 

`this.resultsView?.layout(dimension);`

No need to add 3 extra lines just for an undefined check